### ISSUE TITLE
installment contract report includes voided lines

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1666]
_Imported from trac ticket 1666.
Created by **rneville, sal**
Opened in trac: May 24, 2012, 3:14:24 PM
Last modified in trac: Jul 24, 2012, 9:41:03 AM_
***
_by rneville, sal:_
See SO# 0421243TIUA for contract 0421243O48. This order was created last month but the contract was created yesterday. The description of property is including voided items from the order.

see line 319 of isf_instcontract.html
```
 <tr py:for="index, line in enumerate([l for l in order.orderdetails])" class="${index%2 and 'odd' or 'even'}">
```
needs to be for l in order.nonempty_orderdetails instead

**Comment by kbower** [comment]
_**rneville** trac comment on Jul 24, 2012, 9:41:03 AM:_

(Changeset [[2911]](../commit/107abbb256140a478ab2be62fb6c71ed36b589a4))
 * fixes [#1813](1813) (printing of contract fails if originalcontractid exists on a previously voided 2nd contract)
 * fixes #1666 (installment contract report includes voided lines)
 * fixes [#1824](1824) (contract print fails for refinanced contract when customerid changed)

**Comment by kbower** [comment]
_**rneville** trac status change on Jul 24, 2012, 9:41:03 AM:_ **closed**
